src/controllers/members.js

```diff
@@ -3,14 +3,20 @@ const Member = require('../models/Member');
 const getMemberById = (req, res) => {
   const { id } = req.params;
   Member.findById(id)
```

Comment:
You should check if the ID exist before search in DB

Reply:
Validation of the id added in both the delete and get by id methods